tests/features/homepage.feature

```diff
@@ -98,6 +98,51 @@ Feature: Homepage
     When I am on the homepage
     Then I should see "Mercury poisoning" as the highlighted solution
 
+  Scenario: An event can be highlighted on the homepage
+    Given event content:
+      | title                     | start date       | end date         | state     |
+      | Florentine steak festival | 2021-06-04T20:00 | 2021-06-04T22:00 | validated |
+    And the "Highlighted event" content listing contains:
+      | type    | label                     |
+      | content | Florentine steak festival |
+    And the "Highlighted event" content listing has the following fields:
+      | header text  | We don't serve 'well done' |
+      | link text    | Eat our divine meat        |
+      | external url | http://raresteaktown.com/  |
```

Comment:
The UI is not tested since it is moderator-only and according to our guidelines we should not test this, but feel free to check it out. It can be reached by logging in as a moderator and following User profile > Dashboard > Curated content listings > Highlighted event.